.github/workflows/yaml.yaml

```diff
@@ -4,7 +4,13 @@ name: YAML Validation
 
 on:
   pull_request:
+    paths:
+      - '**.yml'
```

Comment:
Could we have only `yaml` files and no `yml` ones?

Reply:
Yep, there are no `.yml` files in the repo.

Reply:
looks like there are both, yaml and yml files
```
ker@dus:~/github/PHP-CS-Fixer λ git ls | grep yml
.appveyor.yml
.circleci/config.yml
.github/FUNDING.yml
.github/ISSUE_TEMPLATE/config.yml
.travis.yml
```

```
ker@dus:~/github/PHP-CS-Fixer λ git ls | grep yaml
.github/workflows/ci.yaml
.github/workflows/sca.yaml
.github/workflows/yaml.yaml
```